### PR TITLE
SDI-4: ✨ Improve prison name

### PR DESCRIPTION
**Pull request by petergphillips**

### CODE REVIEW

File: src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java [hunk 1]
```diff
@@ -1119,7 +1120,12 @@ private Sort mapBookingSortOrderProperties(Sort sort) {
     }
 
     @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA"})
-    public List<Prison> getBookingVisitsPrisons(final Long bookingId) {
-        return visitInformationRepository.findByBookingIdGroupByPrisonId(bookingId);
+    public List<PrisonDetails> getBookingVisitsPrisons(final Long bookingId) {
+        return visitInformationRepository.findByBookingIdGroupByPrisonId(bookingId)
+            .stream().map(prison -> PrisonDetails.builder()
+                .prisonId(prison.getPrisonId())
+                .prison(LocationProcessor.formatLocation(prison.getPrisonDescription()))
```
Comment (petergphillips) [inline src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java:1127]: format prison name

File: src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java [hunk 2]
```diff
@@ -1119,7 +1120,12 @@ private Sort mapBookingSortOrderProperties(Sort sort) {
...
+                .build())
+            .collect(Collectors.toList());
```
Comment (andrewrlee) [inline src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java:1129]: meh.. don't you get a warning around just using `Stream.toList()` instead?

**Review by andrewrlee — APPROVED** [review]
LGTM

File: src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java (hunk 2 above)
Comment (petergphillips) [reply src/main/java/uk/gov/justice/hmpps/prison/service/BookingService.java:1129]: I didn't but will take a look

**Pull request merged**